### PR TITLE
New RPG rockets, rocket bugfixes

### PR DESCRIPTION
**Pull request by Humonitarian**
## About The Pull Request

Adds Thermobaric (heatwave) rockets, as well as HEAT rockets. Changes the HESH rocket sprite, and adds new sprites for all rockets (except scrap).
Fixes a bug where an explosive hitting a wall with bulletholes on it would explode as many times as there were bulletholes.

## Why It's Good For The Game
More cool rockets with new effects, bugfixes

## Changelog
:cl:
add: Thermobaric and HEAT rockets
fix: Rockets and grenadeshells no longer duplicate forever
imageadd: More rocket icons
/:cl:

### CODE REVIEW

**Comment by thedarkcrusader** [comment]
HESH
image

Thermobaric
image

HE
image

HEAT
image

File: code/modules/projectiles/projectile/force.dm [hunk 1]
```diff
@@ -26,3 +26,35 @@
...
+	damage_types[BRUTE] -= 10
+	if(!jet_range)
+		kill_count = 0
+	..()
+
+/obj/item/projectile/forcebolt/jet/on_hit(atom/movable/target)
+	if(isliving(target))
+		var/throwdir = get_dir(firer,target)
+		target.throw_at(get_edge_target_turf(target, throwdir),4,2,firer)
+	return TRUE
```
Comment (TheShown911) [inline code/modules/projectiles/projectile/force.dm:60]: Newline

**Review by TheShown911 — COMMENTED** [review]
How do you get these rockets?

**Comment by Humonitarian** [comment]
> How do you get these rockets?

Unimplemented, other than adminspawn.

**Comment by Firefox13** [comment]
They need an explanation of their special effects

**Review by hyperioo — COMMENTED** [review]
Seems good but add a few words to the description (heat -> lots of shrapnel + penetration effect of the forcebolt, thermobaric -> tiny explosion with chonky heatwave)

**Review by hyperioo — APPROVED** [review]

**Pull request merged**